src/testing/validation-error-formatter.js

```diff
@@ -186,6 +186,15 @@ exports.mustBeTrimmedViolation = (itemPath) => `item "${itemPath}" must not have
 exports.mustEqualViolation =
   (itemPath, expectedItemValue) => `value of item "${itemPath}" must equal ${JSON.stringify(expectedItemValue)}`;
 
+  /**
```

Comment:
alignment on this top comment block start looks a bit off to me

Reply:
Whoops, looks like I didn't even give ya a chance to correct this!

Reply:
Oh yeah. I'm not sure how that happened.